tests/integration/bundles/kfp_1.7_stable_install.yaml.j2

```diff
@@ -61,6 +61,7 @@ applications:
     charm: kfp-ui
     channel: 2.0/stable
     scale: 1
+    trust: true
```

Comment:
You should add trust one more time because there are two entries of of `kfp-ui` due to an `if` statement.

Reply:
I missed this until just now - I wish I noticed it when you wrote it I wondered why that wasn't working :D